prov/udp/src/udpx_attr.c

```diff
@@ -66,10 +66,10 @@ struct fi_domain_attr udpx_domain_attr = {
 	.resource_mgmt = FI_RM_ENABLED,
 	.av_type = FI_AV_UNSPEC,
 	.mr_mode = FI_MR_SCALABLE,
-	.cq_cnt = (1 << 16),
+	.cq_cnt = 1024,
 	.ep_cnt = (1 << 15),
```

Comment:
updated, but I guess this view doesn't reflect the changes.

I went back and forth on ep_cnt.  The tx_cxt_cnt and rx_ctx_cnt attributes are set to the lower limit.  A better option is to determine this dynamically, but figuring that out seemed like too much work.